src/app/index.js

```diff
@@ -294,6 +303,12 @@ export default yeoman.generators.Base.extend({
 
     this._currentPkgJSON = this.fs.readJSON(this.destinationPath('package.json'), null);
 
+    this._ghooksOptions = {
+      lint: 'Check code quality using videojs-standard',
+      test: 'Run unit tests using Karma',
```

Comment:
Sounds good. And maybe leave out the "using Karma" bit as that's kind of redundant information.